src/v4/base/ReactorStructs.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {IReactor} from "../../interfaces/IReactor.sol";
+import {IPreExecutionHook, IPostExecutionHook} from "../interfaces/IHook.sol";
+import {InputToken, OutputToken} from "../../base/ReactorStructs.sol";
+
+/// @dev generic order information
+///  should be included as the first field in any concrete order types
+struct OrderInfo {
```

Comment:
Why is the `auctionResolver` not included here?

Reply:
maybe makes sense to have it in OrderInfo too, so that we can verify it against user signature